src/alfasim_sdk/_internal/constants.py

```diff
@@ -358,6 +359,11 @@ class MultiInputType(Enum):
     Curve = "curve"
 
 
+class EspParameters(Enum):
+    UserDefined = auto()
+    Catalog = auto()
```

Comment:
As @prusse-martin has commented in Zulip, we should use explicit constants here

```suggestion
class EspParameters(Enum):
    UserDefined = "user_defined"
    Catalog = "catalog"
```